www/main/sttm-ui/switch/Switch.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const Switch = ({ title, controlId, className, onToggle, defaultValue = false }) => {
+  const [isSwitched, toggleSwitchedState] = useState(defaultValue);
```

Comment:
do we want to the state to update if the prop updates? in which case you'll need to `useEffect`